.github/workflows/changesets.yml

```diff
@@ -47,5 +47,9 @@ jobs:
         run: pnpm install --frozen-lockfile
       - name: Create Release Pull Request
         uses: changesets/action@e9cc34b540dd3ad1b030c57fd97269e8f6ad905a # pin@v1
+        with:
+          # Generates src/version.ts file using genVersion
+          # https://github.com/changesets/action#with-version-script
+          version: pnpm run codegen:version
```

Comment:
@Jordan-Mysten do you know if there's a good way to test this change(e.g. verify that this is included in the new Package Version PR) without merging this PR?